src/components/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,34 @@
+import { NavLink } from 'react-router-dom';
+import cn from 'classnames';
+
+export function NavBar() {
+  return (
+    <nav
+      className="navbar is-light is-fixed-top is-mobile has-shadow"
+      data-cy="Nav"
+    >
+      <div className="container">
+        <div className="navbar-brand">
+          <NavLink
```

Comment:
DRY
make a separate component to reuse `isActive`
Check video in mate how to avoid duplicates